File: src/main/java/pl/futurecollars/invoice/model/Company.java
package pl.futurecollars.invoice.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Company {

  private String name;
  private String taxIdentificationNumber;
  private String address;

}

File: angular-frontend/invoicing-app/src/app/company.ts
export class Company {

  public editMode: boolean = false;
  public editedCompany: Company = null;

  constructor(
    public taxIdentificationNumber: string,
    public address: string,
    public name: string,
    public pensionInsurance: number,
    public healthInsurance: number
  ) {
  }
}
